tasks/libs/ciproviders/gitlab.py

```diff
@@ -396,21 +405,141 @@ def generate_gitlab_full_configuration(input_file):
     full_configuration = {}
     for yaml_file in yaml_contents:
         full_configuration.update(yaml_file)
+    # Override some variables with a dedicated context
+    if context:
+        full_configuration["variables"].update(context)
+    if compare_to:
```

Comment:
This option is not used yet, I'm still trying to see if a change in the configuration can work on the gitlab API call